gazelle/python/testdata/per_file_subdirs/bar/BUILD.out

```diff
@@ -11,3 +11,12 @@ py_library(
     srcs = ["foo.py"],
     visibility = ["//:__subpackages__"],
 )
+
+py_test(
+    name = "bar_test",
+    srcs = [
+        "__test__.py",
```

Comment:
Could you please add another test file to see that multiple targets are generated?